pos_sale_order/README.rst

```diff
@@ -0,0 +1,26 @@
+POS To Sale Order
+=================
+
+This module allows to create a sale order instead of a pos order in POS.
```

Comment:
If this module replace pos.order by sale.order, the sentence must be rephrased to this, because allows means a possibility/choice. If there is this possibility, the explain in Usage/Configuration sections how to do it.